workshop/Lab1/README.md

```diff
@@ -11,69 +11,85 @@ If you haven't already:
 
    ```$ bx cs cluster-create --name <name-of-cluster>```
 
-# 1. Push an image to IBM Cloud Container Registry
+Once the cluster is provisioned, the kubernetes client CLI `kubectl` needs to be
+configured to talk to the provisioned cluster.
 
-1. Download a copy of this repository:
-
-   ```$ git clone https://github.com/IBM/kube101```
-
-2. Change directory to Lab 1:
-
-   ```$ cd Lab1```
-
-3. Verify that your Kubernetes cluster is ready for use
-
-   1. Run `$ bx cs clusters` and wait until your cluster is in "normal" state.
-   2. Use `$ bx cs workers <name-of-cluster>`, and make sure that all workers
-      are in "normal" state with "Ready" status.
+1. Run `$ bx cs cluster-config <name-of-cluster>`, and set the `KUBECONFIG`
+   environment variable based on the output of the command. This will
+   make your `kubectl` client point to your new Kubernetes cluster.
 
-You are now ready to use Kubernetes to deploy an application, in this case
-the "guestbook" application.
+Once your client is configured, you are ready to deploy your first application, `guestbook`.
 
-# 2. Deploy your application
+# 1. Deploy your application
 
 In this part of the lab we will deploy an application called `guestbook`
 that has already been built and uploaded to DockerHub under the name
 `ibmcom/guestbook:v1`.
 
-1. Run `$ bx cs cluster-config <name-of-cluster>`, and set the `KUBECONFIG`
-   environment variable based on the output of the command. This will
-   make your `kubectl` client point to your new Kubernetes cluster.
-
-2. Start by running `guestbook` as a deployment:
+1. Start by running `guestbook`:
 
    ```$ kubectl run guestbook --image=ibmcom/guestbook:v1```
 
-   This action will take a bit of time. To check the status of your deployment,
+   This action will take a bit of time. To check the status of the running application,
    you can use `$ kubectl get pods`.
 
    You should see output similar to the following:
 
-   ```
+   ```console
    $ kubectl get pods
    NAME                          READY     STATUS              RESTARTS   AGE
    guestbook-59bd679fdc-bxdg7    0/1       ContainerCreating   0          1m
    ```
-
+   Eventually, the status should show up as `Running`.
+   
+   ```console
+   $ kubectl get pods
+   NAME                          READY     STATUS              RESTARTS   AGE
+   guestbook-59bd679fdc-bxdg7    1/1       Running             0          1m
+   ```
+   
+   The end result of our the run command is not just the pod containing our application containers,
+   but a Deployment resource that manages the lifecycle of those pods.
+   We can look at the details of the deployment with `kubectl describe deployment guestbook`.   
```

Comment:
Do we want to talk about what they'll see? If we do it later then add something like "We'll discuss the details of this command's output later".

We also may want to be consistent and show all commands we want them to type on new lines, instead of embedded within paragraphs, like this one.   We could then create a script to extract all command for auto-testing.

Reply:
not required to mention describe command. Taking it out.